compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/LangLibrary.java

```diff
@@ -140,7 +140,8 @@ public static LangLibrary getInstance(CompilerContext context) {
 
     private void populateMethodList(List<FunctionSymbol> list, Map<String, BInvokableSymbol> langLib) {
         for (Map.Entry<String, BInvokableSymbol> entry : langLib.entrySet()) {
-            FunctionSymbol method = symbolFactory.createFunctionSymbol(entry.getValue(), entry.getKey());
+            String name = entry.getValue().getOriginalName().getValue();
+            FunctionSymbol method = symbolFactory.createFunctionSymbol(entry.getValue(), name);
```

Comment:
If the key is not required, can consider looping through `values()` of the map, instead of the entry set.